pyproject.toml

```
@@ -1,3 +1,7 @@
+[[tool.mypy.overrides]]
+ignore_missing_imports = true
+module = ["pytest_venv"]
```

If we keep this, then I'd vote for splitting like this like the rest of the file. Means shorter diffs if we add to it and consistency.
```suggestion
module = [
	"pytest_venv",
]
```

---

Have went with suggestion for inline ignore as seemed cleaner so marking this as resolved.